src/components/CameraFeed/ConfigureCamera.tsx

```diff
@@ -389,16 +389,37 @@ export default function ConfigureCamera(props: Props) {
                     setPresetName("");
                   }}
                 >
-                  <TextFormField
-                    name="preset-name"
-                    className="py-4"
-                    value={presetName}
-                    onChange={({ value }) => setPresetName(value)}
-                    errorClassName="hidden"
-                    placeholder={t("preset_name_placeholder")}
-                    suggestions={presetNameSuggestions}
-                  />
-                  <div className="cui-form-button-group">
+                  <div className="relative w-full">
+                    <TextFormField
+                      name="preset-name"
+                      className="w-full py-4"
+                      value={presetName}
+                      onChange={({ value }) => setPresetName(value)}
+                      errorClassName="hidden"
+                      placeholder={t("preset_name_placeholder")}
+                      suggestions={presetNameSuggestions}
+                    />
+                    {presetName && (
+                      <button
+                        type="button"
+                        className="absolute right-2 top-1/2 -translate-y-1/2 transform text-gray-500"
+                        onClick={() => setPresetName("")}
+                      >
+                        <CareIcon icon="l-times-circle" className="text-lg" />
+                      </button>
+                    )}
+                  </div>
+
+                  <div className="cui-form-button-group mt-4 flex w-full flex-col justify-end space-y-2 sm:flex-row sm:space-x-2 sm:space-y-0">
+                    <Cancel
+                      shadow={false}
```

Comment:
```suggestion
```

Reply:
Done the changes. I thought to maintain a simpler and cleaner design.